autoload/ctrlsf/win.vim

```diff
@@ -145,6 +145,8 @@ endf
 " SetModifiable()
 "
 func! ctrlsf#win#SetModifiableByViewMode(modifiable) abort
+    call setbufvar(s:MAIN_BUF_NAME, '&modifiable', 1)
+    call setbufvar(s:MAIN_BUF_NAME, '&fileencoding', g:ctrlsf_encoding)
```

Comment:
To make things simpler, I prefer keeping ctrlsf buffer's encoding to `utf-8`. You could iconv characters in other encoding into utf8 during rendering.